.github/labels.md

```diff
@@ -0,0 +1,234 @@
+an issue needs further interaction and who should carry out this interaction.
+Issue have exactly one status.
+
+
+## Labels
+
+This section lists when an label should be applied and what actions may be taken
+additionally.
+
+
+#### Label `category:bug`
+
+The Issue describes or solves a percieved malfunction within the game.
+
+#### Label `category:feature`
+
+The Pull Request contains a new game feature.  Features are parts of the game
+which a player can interact with e.g.: A new species, a new gameplay mechanic
+or a new command line switch for the server that changes gameplay behaviour.
+
+#### Label `category:idea`
+
+??? No idea, overlaps with `category:feature` or `state:work in progress` IMO.
+
+#### Label `category:question`
+
+??? Questions don't belong into the issue tracker IMO.
+
+#### Label `category:refactoring`
+
+The Pull request contains a new or improved implementation of an existing
+feature or any other code restructuring that doesn't modify the current game
+mechanics or representation.
+
+#### Label `category:request`
+
+??? Questions don't belong into the issue tracker IMO.
+
+#### Label `category:tweak`
+
+??? No idea.
+
+#### Label `component:AI`
+
+The Issue requires knowledge about the python AI decision making code or affects
+it.
+
+#### Label `component:art music text assets`
+
+The Issue requires knowledge about creating game assets or verification if the
+assets match the style goals of the game.
+
+#### Label `component:build system`
+
+The Issue requires knowledge about CMake, Visual Studio, Xcode or the general
+build process.
+
+#### Label `component:common`
+
+??? No idea.
+
+#### Label `component:deployment`
+
+The Issue requires knowledge about the deployment process or the software used
+create the installer. ??? I see an overlap with `component:build system` here.
+
+#### Label `component:documentation`
+
+The Issue requires knowledge about Doxygen or primarly contains documentation.
+
+#### Label `component:game mechanic`
+
+The Pull Request requires knowledge about the currently used or planned game
+mechanics.  The content of the Pull Request may require extensive gameplay
+balance testing.
+
+#### Label `component:infrastructure`
+
+The Pull Request requires knowledge about non-game related infrastructure that
+aids development.  For example this includes issues with the project homepage,
+used external services like continous integration or hosting of release
+binaries.
+
+#### Label `component:internal`
+
+??? No idea.  Thread suggest it's a non-user visible feature, but then it's the
+wrong category.
+
+#### Label `component:network`
+
+The Issue requires knowledge about TCP/IP networking and establishing a test
+network infrastructure.
+
+#### Label `component:scripting`
+
+The Pull Request requires knowledge about the FOCS language and probably game
+mechanics.  Probably most used in conjunction with `component:game mechanic` and
+`component:art music text assets`.
+
```

Comment:
Not an objection, but if "scripting" is changed to "FOCS", we need another label for the universe generation and turn event Python scripts, as they are covered neither by "AI" nor by "FOCS".